pkg/provisioner/ironic/updateopts_test.go

```diff
@@ -286,3 +286,73 @@ func TestGetUpdateOptsForNodeDell(t *testing.T) {
 		})
 	}
 }
+
+func TestGetUpdateOptsForNodeLiveIso(t *testing.T) {
+	eventPublisher := func(reason, message string) {}
+	auth := clients.AuthConfig{Type: clients.NoAuth}
+
+	prov, err := newProvisionerWithSettings(makeHostLiveIso(), bmc.Credentials{}, eventPublisher,
+		"https://ironic.test", auth, "https://ironic.test", auth,
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+	ironicNode := &nodes.Node{}
+
+	patches, err := prov.getUpdateOptsForNode(ironicNode)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Logf("patches: %v", patches)
+
+	expected := []struct {
+		Path  string      // the node property path
+		Key   string      // if value is a map, the key we care about
+		Value interface{} // the value being passed to ironic (or value associated with the key)
+	}{
+		{
+			Path:  "/instance_info/boot_iso",
+			Value: "not-empty",
+		},
+		{
+			Path:  "/deploy_interface",
+			Value: "ramdisk",
+		},
+		{
+			Path:  "/instance_uuid",
+			Value: "27720611-e5d1-45d3-ba3a-222dcfaa4ca2",
```

Comment:
Ah, so it's not missing. Great.

Reply:
It's actually set in two places which is why this test didn't catch my removal - I'll look at improving the test coverage as a follow-up.

Reply:
To close the loop on this, it's no longer set in two places after https://github.com/metal3-io/baremetal-operator/commit/177d9efb4d627f6c3d9fae92e0b68c94c06aa713